loom/graphics/gfxVectorGraphics.cpp

```diff
@@ -148,6 +148,7 @@ void VectorGraphics::lineStyle(float thickness, unsigned int color, float alpha,
 void VectorGraphics::textFormat(VectorTextFormat format) {
 	ensureTextFormat();
 	queue->push_back(lmNew(NULL) VectorTextFormatData(lmNew(NULL) VectorTextFormat(format)));
+	currentTextFormat = format;
```

Comment:
That would be `clear()`, but that resets `textFormatDirty`,  which in turn also resets the format eventually.